packages/commonwealth/client/styles/mixins/text.scss

```diff
@@ -26,7 +26,7 @@
   font-family: $font-family-neue-haas-unica;
   font-size: 12px;
   letter-spacing: 0.02em;
-  line-height: 18px;
```

Comment:
this is a global change for whole project for text type `caption`. Do we want this?

Reply:
I changed this based on updates to the typography figma.